spec/forms/new_phone_form_spec.rb

```diff
@@ -247,8 +245,6 @@
       end
 
       context 'when voip numbers are blocked' do
```

Comment:
I think we could remove / flatten this context block wrapper, since it's the default behavior..